aten/src/ATen/native/quantized/cpu/qconv_unpack.cpp

```diff
@@ -204,6 +204,15 @@ class QConvPadding final {
   }
 };
 
+template <int kSpatialDim = 2>
+class QConvOutputPadding final {
+ public:
+  static torch::List<int64_t> run(
+      const c10::intrusive_ptr<ConvPackedParamsBase<kSpatialDim>>& packed_weight) {
```

Comment:
nit: `packed_params`, since it's actually not the weight that's being passed in?

Reply:
Although we call it `ConvPackedParams`, I think on python level it is still called `packed_weight`. I can change it to the `params` -- I am just concerned if this would be equally confusing?